File: Game Jam 3/Assets/TankMove.js
#pragma strict

var moveSpeed : float = 100.0f;
var isMultiplayer : boolean = false;

function Start () {

}

function Update () {
	var translation : float = Input.GetAxis("Vertical") * Time.deltaTime * moveSpeed;
	//Debug.Log("TankMove() >>> Input.GetAxis = ", Input.GetAxis("Vertical");
	//if (Input.GetButtonDown("Fire1")) {
		transform.Translate(Vector3(0.0, translation, 0.0));
	//}
}

File: Tank1/Assets/TankMove.cs
using UnityEngine;
using System.Collections;

public class TankMove : MonoBehaviour {

	public int i = 1;
	public float move = 10.0f;
	// Update is called once per frame
	void Update () {
		float h = Input.GetAxis("Horizontal") * Time.deltaTime * move;
		float v = Input.GetAxis("Vertical") * Time.deltaTime * move;
		transform.Translate(new Vector3(0, 0, v));
	}
}
